macos/ReactTestApp/AppDelegate.swift

```diff
@@ -11,6 +11,13 @@ import Cocoa
 final class AppDelegate: NSObject, NSApplicationDelegate {
     private(set) lazy var reactInstance = ReactInstance()
 
+    private var mainWindow: NSWindow? {
+        // `keyWindow` might be `nil` while loading or when the window is not
+        // active. Use `frameAutosaveName` to find our main window.
+        let windows = NSApplication.shared.windows
+        return windows.first { $0.frameAutosaveName == "MainWindow" }
+    }
```

Comment:
It seems like a better idea to either hookup the window from the nib/storyboard to the `AppDelegate`; _or_, if the nib/storyboard doesn’t do anything of interest, just remove the window from there and create it programmatically instead.

Reply:
Sounds good, be sure to file an issue and link it here please 🙏

Reply:
Sure thing: #111